definitions/variable/energy/emission-intensity.yaml

```diff
@@ -0,0 +1,107 @@
+- Emission Intensity|CO2|Industry|Iron and Steel:
+    description: Emission intensity of direct energy-related CO2 (within Scope 1) per output from the Iron and Steel industry
+    unit: Mt CO2/Mt
+    tier: 2
+- Emission Intensity|CO2|Industry|Iron and Steel({scope}):
+    description: Emission intensity of {scope} CO2 per output from the Iron and Steel industry
+    unit: Mt CO2/Mt
+    tier: 3
+
+- Emission Intensity|CO2|Industry|Non-Metallic Minerals|{Non-Metallic Minerals Commodity}:
+    description: Emission intensity of direct energy-related CO2 (within Scope 1) per output from the {Non-Metallic Minerals Commodity} industry
+    unit: Mt CO2/Mt
+    tier: 2
+- Emission Intensity|CO2|Industry|Non-Metallic Minerals|{Non-Metallic Minerals Commodity}({scope}):
+    description: Emission intensity of {scope} CO2 per output from the {Non-Metallic Minerals Commodity} industry
+    unit: Mt CO2/Mt
+    tier: 3
+
+- Emission Intensity|CO2|Industry|Non-Ferrous Metals|{Non-Ferrous Metals Commodity}:
+    description: Emission intensity of direct energy-related CO2 (within Scope 1) per output from the {Non-Ferrous Metals Commodity} industry
+    unit: Mt CO2/Mt
+    tier: 2
+- Emission Intensity|CO2|Industry|Non-Ferrous Metals|{Non-Ferrous Metals Commodity}({scope}):
+    description: Emission intensity of {scope} CO2 per output from the {Non-Ferrous Metals Commodity} industry
+    unit: Mt CO2/Mt
+    tier: 3
+
+- Emission Intensity|CO2|Industry|Chemicals|{Chemicals Commodity}:
+    description: Emission intensity of direct energy-related CO2 (within Scope 1) per output from the {Chemicals Commodity} industry
+    unit: Mt CO2/Mt
+    tier: 2
+- Emission Intensity|CO2|Industry|Chemicals|{Chemicals Commodity}({scope}):
+    description: Emission intensity of {scope} CO2 per output from the {Chemicals Commodity} industry
+    unit: Mt CO2/Mt
+    tier: 3
+
+- Emission Intensity|CO2|Transportation|Passenger|Road|Light-Duty Vehicle:
+    description: Emission intensity of direct energy-related CO2 (within Scope 1) per output from the passenger transport on roads in light duty vehicles
```

Comment:
Please replace "output" by the correct metric - I guess passenger-kilometers traveled?

Reply:
"energy service demand" is still difficult to understand for non-experts, and the unit is pkm, not energy (service). So why not write "per passenger-distance travelled in light duty vehicles"?